frontend/views/components/TextWho.vue

```diff
@@ -6,7 +6,7 @@
       <tooltip>
         <button type="button" class="gi-is-unstyled gi-is-link-inherit">{{who.length - 1}}<i18n>others</i18n></button>
         <template slot="tooltip">
-          <p v-for="(name, index) in who" v-if="index > 0">{{name}}</p>
+          <p v-for="(name, index) in who.shift()" :key="`name-${index}`">{{name}}</p>
```

Comment:
This directly modifies the `who` variable, which could cause problems? Or at least it's not clear then what the value of `who.length - 1` is above.

Can this be rewritten without modifying `who` directly, maybe on a copy (using `slice`), or rewritten in some other way?